Makefile

```diff
@@ -29,8 +29,8 @@ MOCK_CONFIG=default
 all:
 	@echo
 	@echo "Development related targets:"
-	@echo "check:      Runs tree static check, unittests and functional tests"
-	@echo "check-long: Runs tree static check, unittests and long functional tests"
+	@echo "check:      Runs tree static check, unittests and fast functional tests"
```

Comment:
well they are fast and medium-fast 😄, but the `check-long` to `check-full` is the most important matter and the description is understandable, so let's go with this...